resources/tests/test_admin_actions.py

```diff
@@ -0,0 +1,29 @@
+# -*- coding: utf-8 -*-
+import pytest
+from django.contrib.auth import get_user_model
+
+from resources.models import Reservation
+from users.admin import anonymize_user_data
+
+
+@pytest.mark.django_db
+def test_anonymize_user_data(api_client, resource_in_unit, user):
+    """
+    Test anonymization of user data.
+    """
+    user.first_name = 'testi_ukkeli'
+    user.save()
+    Reservation.objects.create(
+        resource=resource_in_unit,
+        begin='2015-04-04T09:00:00+02:00',
+        end='2015-04-04T10:00:00+02:00',
+        user=user,
+        reserver_name='John Smith',
+        event_subject="John's welcome party",
+        state=Reservation.CONFIRMED
+    )
+    # anonymize_user_data expects a queryset instead of single object
+    test_user = get_user_model().objects.all().filter(first_name='testi_ukkeli')
+    anonymize_user_data(modeladmin=None, request=None, queryset=test_user)
+    assert get_user_model().objects.filter(first_name='testi_ukkeli').count() == 0
+    assert Reservation.objects.filter(resource=resource_in_unit)[0].event_description == 'Sensitive data of this reservation has been anonymized by a script.'
```

Comment:
I think `.filter(resource=resource_in_unit)[0]` could be replaced with `.get(resource=resource_in_unit)`

Reply:
Line length here is also too long (155 chars) for pep8 spec (max 120 chars). Could do something like:

`anonymized_field_text = 'Sensitive data of this reservation has been anonymized by a script.'
assert Reservation.objects.get(resource=resource_in_unit).event_description == anonymized_field_text`